cl/assets/static-global/js/react/vendor.js.map

```diff
@@ -1 +0,0 @@
-{"version":3,"file":"vendor.js","sources":["webpack:///vendor.js"],"mappings":"AAAA;;;;;AA8ZA;AAyoCA;;;;AA8sKA;;;;;AAsvaA;;;;;;;AAy/FA;;;;;;;;AAkiRA;;;;;;;;AAmCA;;;;;;;;AAoTA;;;;;;;;AA28BA;;;;;;;;AA0YA;;;;;;;;AAg7DA;;;;;;;;AAkCA","sourceRoot":""}
```

Comment:
I’m not sure why `sources` is only `["webpack:///vendor.js"]` (or how to generate _this_ output). The good news is that it means this source map is essentially meaningless – and harmless to remove.